ClosedXML.Sandbox/ClosedXML.Sandbox.csproj

```diff
@@ -1,7 +1,7 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
 
   <PropertyGroup>
-    <TargetFrameworks>netcoreapp3.1;net40;net46</TargetFrameworks>
+    <TargetFrameworks>netcoreapp3.1;net461</TargetFrameworks>
```

Comment:
net461 is EOL (Apr 26, 2022), would it be preferable to target supported frameworks, being net462 ?